src/Compilers/CSharp/Test/Emit/Emit/LocalStateTracing/LocalStateTracingTests.cs

```diff
@@ -0,0 +1,5898 @@
+    IL_0058:  nop
+    IL_0059:  endfinally
+  }
+  // sequence point: }
+  IL_005a:  ret
+}
+");
+        }
+
+        [Fact]
+        public void Initializers()
+        {
+            var source = WithHelpers(@"
+class C
+{
+    static int A = F(out var x) + (x = 1);
+    static int B = F(out var x) + (x = 2);
+
+    static int F(out int a) => a = 1;
+}
+");
+            var verifier = CompileAndVerify(source);
+            verifier.VerifyMethodBody("C..cctor", @"
+{
+  // Code size       95 (0x5f)
+  .maxstack  4
+  .locals init (Microsoft.CodeAnalysis.Runtime.LocalStoreTracker V_0,
+                int V_1, //x
+                int V_2) //x
+  // sequence point: <hidden>
+  IL_0000:  ldtoken    ""C..cctor()""
+  IL_0005:  call       ""Microsoft.CodeAnalysis.Runtime.LocalStoreTracker Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogMethodEntry(int)""
+  IL_000a:  stloc.0
+  .try
+  {
+    // sequence point: static int A = F(out var x) + (x = 1);
+    IL_000b:  ldloca.s   V_1
+    IL_000d:  call       ""int C.F(out int)""
+    IL_0012:  ldloca.s   V_0
+    IL_0014:  ldloc.1
+    IL_0015:  ldc.i4.1
+    IL_0016:  call       ""void Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogLocalStore(uint, int)""
+    IL_001b:  nop
+    IL_001c:  ldloca.s   V_0
+    IL_001e:  ldc.i4.1
+    IL_001f:  dup
+    IL_0020:  stloc.1
+    IL_0021:  ldc.i4.1
+    IL_0022:  call       ""void Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogLocalStore(uint, int)""
+    IL_0027:  nop
+    IL_0028:  ldloc.1
+    IL_0029:  add
+    IL_002a:  stsfld     ""int C.A""
+    // sequence point: static int B = F(out var x) + (x = 2);
+    IL_002f:  ldloca.s   V_2
+    IL_0031:  call       ""int C.F(out int)""
+    IL_0036:  ldloca.s   V_0
+    IL_0038:  ldloc.2
+    IL_0039:  ldc.i4.2
+    IL_003a:  call       ""void Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogLocalStore(uint, int)""
+    IL_003f:  nop
+    IL_0040:  ldloca.s   V_0
+    IL_0042:  ldc.i4.2
+    IL_0043:  dup
+    IL_0044:  stloc.2
+    IL_0045:  ldc.i4.2
+    IL_0046:  call       ""void Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogLocalStore(uint, int)""
+    IL_004b:  nop
+    IL_004c:  ldloc.2
+    IL_004d:  add
+    IL_004e:  stsfld     ""int C.B""
+    IL_0053:  leave.s    IL_005e
+  }
+  finally
+  {
+    // sequence point: <hidden>
+    IL_0055:  ldloca.s   V_0
+    IL_0057:  call       ""void Microsoft.CodeAnalysis.Runtime.LocalStoreTracker.LogReturn()""
+    IL_005c:  nop
+    IL_005d:  endfinally
+  }
+  // sequence point: <hidden>
+  IL_005e:  ret
+}");
+        }
+
+        [Fact]
+        public void Initializers_Lambda()
+        {
+            var source = WithHelpers(@"
+using System;
+
+class C
+{
+    static Action A = new Action(() => { int x = 1; });
+
+    static void Main() { A(); }
+}
+");
+
+            // cctor invocation timing is different between .NET Core and .NET FX
+#if NET
```

Comment:
Unnecessary since the whole file is inside `#if NET`.

Reply:
Removed. I attempted to make these work for NET FX and gave up.